cmd/minishift/cmd/profile/profile.go

```diff
@@ -17,12 +17,14 @@ limitations under the License.
 package profile
 
 import (
+	"strings"
+
 	"github.com/minishift/minishift/pkg/util/os/atexit"
 	"github.com/spf13/cobra"
 )
 
 const (
-	emptyProfileMessage  = "You must provide the profile name. Run `minishift profile list` to view profiles"
+	emptyProfileMessage  = "A profile name must be provided. Run 'minishift profile list' for a list of existing profiles"
```

Comment:
full stop after 'profiles'. Please also add a full stop after the other error message.